### PR TITLE
BE-2-add-dockerfile

### PR DESCRIPTION
**Pull request by Randayy**

### CODE REVIEW

**Comment by Randayy** [comment]
read me will be in half an hour

**Comment by Randayy** [comment]
image

File: Dockerfile [hunk 1]
```diff
@@ -0,0 +1,10 @@
+FROM python:3.11.5
```
Comment (Ghibliliker) [inline Dockerfile]: alpine

File: Dockerfile [hunk 2]
```diff
@@ -0,0 +1,10 @@
...
+
+
+COPY requirements.txt .
+
+RUN pip install -r requirements.txt
+WORKDIR /app
```
Comment (Ghibliliker) [inline Dockerfile]: it shoud be above

File: app/main.py [hunk 3]
```diff
@@ -1,13 +1,26 @@
 from fastapi import FastAPI
 from .routers.health_check_route import router
-from .core.config import settings
+from .core.config import Settings
```
Comment (Ghibliliker) [inline app/main.py]: full path

File: app/main.py [hunk 4]
```diff
@@ -1,13 +1,26 @@
...
+from fastapi.middleware.cors import CORSMiddleware
 
+settings = Settings()
```
Comment (Ghibliliker) [inline app/main.py:6]: init it in config

File: Dockerfile
Comment (Ghibliliker) [inline Dockerfile:1]: add docker compose file

Comment (Artemooon) [reply Dockerfile:1]: I think we do not need docker-compose in the second task, it can be done in the task number 3

File: app/main.py (hunk 4 above)
Comment (Randayy) [reply app/main.py:6]: Mark said that it is better to init it there

File: Dockerfile
Comment (Randayy) [reply Dockerfile:1]: I will do it in 3rd task

File: Dockerfile (hunk 1 above)
Comment (Randayy) [reply Dockerfile]: fixed

**Review by Artemooon — APPROVED** [review]

**Pull request merged**